bin/tests/it/cmd_line_env.rs

```diff
@@ -10,6 +10,8 @@ use std::path::Path;
 use std::process::{Command, Output, Stdio};
 use std::str::from_utf8;
 use std::sync::{Arc, Condvar, Mutex};
+#[cfg(any(target_os = "windows", target_os = "linux"))]
```

Comment:
Is there a "not mac" option?

Reply:
I honestly just did what the linter suggested, but probably would have been the way to go